tests/Selector/NamedSelectorTest.php

```diff
@@ -162,6 +193,56 @@ public function getSelectorTests()
         );
     }
 
+    public function getLateRegisteredReplacements()
+    {
+        // Fixture file,  selectors,  replacements, selector name,  locator,  expected number of exact matched elements, expected number of partial matched elements if different
+        return array(
```

Comment:
With all these replacements in place, I'm not clearly seeing from the test code what each fixture should find (what the final selector for search should look like). 

Please add some comments about each fixture with selectors .